swyft/utils/utils.py

```diff
@@ -145,8 +145,9 @@ def estimate_coverage(
     diags = []
     for _ in range(nrounds):
         for point in dataset:
-            samples = marginals(point["obs"], nsamples)
-            diag = sample_diagnostics(samples, true_params=point["par"])
+            samples = post.sample(nsamples, point[0])
+            diag = sample_diagnostics(samples, true_params=point[2])
+            print(diag)
```

Comment:
Does it make sense to have a print statement?

Reply:
I just tested it and I don't think it makes sense.

Reply:
Same, wrong function